src/app/core/store/collections/collections.action.ts

```diff
@@ -219,7 +219,7 @@ export namespace CollectionsAction {
   export class AddFavoriteFailure implements Action {
     public readonly type = CollectionsActionType.ADD_FAVORITE_FAILURE;
 
-    public constructor(public payload: { error: any }) {
+    public constructor(public payload: { collectionId: string, error: any }) {
```

Comment:
Why do we still need these actions? Is it not updated with the collection?

Reply:
Of course not. We store information about favorite collections and documents in separate collections, because it's individual for each user.

Reply:
Oh, I see. But we return this information from the back-end inside collection/document object, right?

Reply:
Yes :)